app/models/set_member_subject.rb

```diff
@@ -4,7 +4,6 @@ class SetMemberSubject < ActiveRecord::Base
   
   belongs_to :subject_set, counter_cache: true, touch: true
   belongs_to :subject
-  has_many :classifications
```

Comment:
I think the main issue with losing this relation is that it had the classification counter cache on it. The plan was to use this to cache the number of raw classification for a SMS that was then to be used in simple retirement (first version). If we lost the counter cache then we'll have to query the classification table (possibly very large) and possibly costly as time goes on. 

We can just keep this relation and get all `set_member_subjects` for a `subject_set` and then sum the classification counts. Thoughts?

Finally if we do remove it, then the _count field in the schema should go to.